doc/nrf/ug_partition_manager.rst

```diff
@@ -322,3 +322,75 @@ CMake usage
+
+Static configuration defines the static partitions.
+The area used by the static partitions is called the **static area**.
+Using static configuration, it is still allowed to configure partitions inside the **dynamic area**.
+The dynamic area consists of the "app" partition and all memory adjacent to the "app" partition which is not occupied by a static partition.
+Note that there exists only one dynamic area.
+When Partition Manager is executed, it only operates on the dynamic area, assuming that all other memory is reserved.
+Within the dynamic area, it is allowed to define new partitions.
+The dynamic area is re-sized by removing partitions from or adding partitions to the static configuration.
+
+.. _ug_pm_static_providing:
+
+Providing static configuration
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+The static configuration is provided through a YAML formatted configuration file.
+This file is similar to the regular :file:`pm.yml` configuration files, except
+that it also defines the start address for all partitions.
+The given configuration for a build can be found in :file:`${BUILD_DIR}/partitions.yml`.
+To apply the current configuration as a static configuration, copy this file to `${APPLICATION_SOURCE_DIR}/pm_static.yml``.
+
+.. note::
+  Configuration of the `app` partition inside the static configuration is ignored.
+
+When the build system sees a file named :file:`pm_static.yml` in an applications source directory, it is automatically provided to the partition manager script as the static configuration.
+If desired, the file can be modified as described below.
+
+.. _ug_pm_static_remove:
+
+Removing a static partition
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+A static partition is removed by deleting its entry in :file:`pm_static.yml`.
+Only partitions adjacent to the ``app`` partition or other removed partitions can itself be removed.
+
+.. _ug_pm_static_add_dynamic:
+
+Adding a dynamic partition
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+New dynamic partitions are added automatically by the build process if they are listed in a :file:`pm.yml` file, unless also listed in the static configuration.
+
+.. note::
+   When resolving the relative placement of dynamic partitions any placement
+   properties referencing static partitions will be ignored.
+
+.. _ug_pm_static_add:
+
+Adding a static partition
+~~~~~~~~~~~~~~~~~~~~~~~~~
+To add a new static partition, add an entry for it in the file :file:`pm_static.yml`.
+This entry must define the properties ``address``, ``size``, and if applicable - ``span``.
+
+.. code-block:: yaml
+   :caption: Example of static configuration of partition with span.
+
+   partition_name:
+      address: 0xab00
+      size: 0x1000
+      span: [example]  # Only if this partition had the `span` property set originally.
+
+.. note::
+  Sub-images with the built strategy `*_BUILD_STRATEGY_SKIP_BUILD` or
+  `*_BUILD_STRATEGY_USE_HEX_FILE` must define a static partition to ensure correct
+  placement of the dynamic partitions.
```

Comment:
What about images that define static partitions, do they need to enable _SKIP_BUILD or _USE_HEX_FILE ?

E.g. is it "a if b", or "a iff b"

Reply:
No, you are free to have any build strategy for static partitions, only their placement and size is set.